server/graphql/schema.ts

```diff
@@ -53,6 +58,21 @@ const typeDefs = gql`
         requestGroup(id: ID): RequestGroup
         requestGroups: [RequestGroup]
     }
+    type Mutation {
+        createRequestType(requestType: RequestTypeInput): ServerResponse
```

Comment:
In the schema the type of requestType is RequestTypeInput, but in the datasource the type is RequestTypeDocument (which basically is the RequestTypeInterface). I was wondering if this would cause any problems.